resources/compatibility.md

```diff
@@ -5,8 +5,110 @@ shortname: Resources
 title: Browser Compatibility
 ---
 
+<style>
```

Comment:
`<style>` elements are only allowed in the `<head>` (or shadow root). Can this be moved to a stylesheet that's included at the top?